gnovm/stdlibs/chain/chain.gno

```diff
@@ -0,0 +1,80 @@
+// Package chain contains core types and their helper functions to deal with the
+// blockchain.
+package chain
+
+import (
+	iruntime "internal/runtime"
+)
+
+// Address represents a bech32-encoded blockchain address.
+type Address string
+
+func (a Address) IsValid() bool {
+	// TODO: make stdlib crypto/bech32
+	_, _, ok := DecodeBech32(string(a))
+	return ok
+}
+
+func NewAddressFromImportPath(importPath string) {
+	// TODO: find better name
+	panic("not implemented")
+}
+
+// A Realm represents an on-chain entity which has an address. This is generally
+// an Externally Owned Account (ie. a "user") or a code realm, which also has an
+// associated import path.
+type Realm struct {
+	address    Address
+	importPath string
```

Comment:
What's the problem with `PkgPath`?

Reply:
I am for a different name that pkgpath, because it conflicts at times when explaining the topics. possibly `URL`, `URI`, `ID`, just `path`, etc. in the end its a unique identifier for code, and it can be used to look up code or import code.
- realm path - better than realm package path
- pure package path - better than pure package package path.